bitnami/jenkins/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: jenkins
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/jenkins
   - https://jenkins.io/
-version: 12.1.1
+version: 12.2.0
```

Comment:
Could you please bump a patch version? `12.1.2` should be fine.

Our charts follow [semver](https://semver.org/) versioning: majors for non-backward compatible changes, minors for new features and capabilities, and patches for bug fixes.